README.md

```diff
@@ -2,24 +2,13 @@
 
 Use Pull Request description as Squash and Merge commit message
 
-[Chrome](https://chrome.google.com/webstore/detail/kgabfelbdpeifcemndligpdfddhbbmfm/)
+This is a fork of [zachwhaley/squashed-merge-message](https://github.com/zachwhaley/squashed-merge-message) that works on any host named `github.*` and not only `github.com`
```

Comment:
I don't think we need to change the README.

Once the PR is merged, I'll update the existing downloads

Reply:
Oops! That was meant to only appear on my fork and not this PR.